tests/scout_apm_tests/api/rename_transaction_test.py

```diff
@@ -0,0 +1,12 @@
+import scout_apm.api
+from scout_apm.core.tracked_request import TrackedRequest
+
+
+def test_rename_transaction():
+    tr = TrackedRequest.instance()
+    assert('transaction_name' not in tr.tags)
```

Comment:
no brackets, though `black` will sort that for you, also shouldn't that be `transaction.name` with a dot?

Reply:
yes, it should be